glide.yaml

```diff
@@ -0,0 +1,19 @@
+package: github.com/giantswarm/draughtsman
+import:
+- package: github.com/giantswarm/microkit
+  version: ef2f8fdba32a0a0275d739abf84371e36880ecdc
```

Comment:
Please use master for the deps unless there is a special reason to pin it. In case of viper or some other weird things please place a comment.